problems/arrayMethods.js

```diff
@@ -36,7 +54,17 @@ const plusTen = () => {};
  * @returns {string[]} All strings remain, all non strings are now empty strings.
  */
 
-const stringsOnly = () => {};
+const stringsOnly = (items) => {
+  return items.map((el) => {
+    if(typeof el !== "string"){
+      let el = ""
+      return el
+    } else {
+      return el
+    }
+  })
+  return items
```

Comment:
Never reach this line of code, nor do we want to. We're already returning the new array on line 58.